eng/ApiCompatExcludeAttributes.txt

```diff
@@ -2,3 +2,4 @@ T:System.Diagnostics.CodeAnalysis.MemberNotNullAttribute
 T:System.Diagnostics.DebuggerGuidedStepThroughAttribute
 T:System.Runtime.CompilerServices.EagerStaticClassConstructionAttribute
 T:System.Runtime.CompilerServices.NativeIntegerAttribute
+T:System.Runtime.CompilerServices.SkipLocalsInitAttribute
```

Comment:
Why wasn't this excluded before? Or to rephrase, what makes System.IO.Hashing special that it requires this attribute exclusion?

Reply:
Everywhere else we use it at the module level.  This is the first time we've used it at the class level.

Reply:
Why can't we apply it for the entire System.IO.Hashing module?

Reply:
We could, it's just broader allocation than we've typically done outside of netcoreapp where this attribute is automatically added at to every module. Regardless, it makes sense to invite the attribute here; it should be in ref assemblies.